specification/communication/data-plane/Identity/stable/2022-10-01/CommunicationIdentity.json

```diff
@@ -339,14 +346,17 @@
       "properties": {
         "token": {
           "description": "Azure AD access token of a Teams User to acquire a new Communication Identity access token.",
+          "minLength": 1,
```

Comment:
Swashbuckle started generating these recently and we agreed that the values make sense and that we'd rather keep them than trying to work around Swashbuckle. 
On requests we return BadRequest if these values are missing.

Reply:
What happened in previous versions of your service?  Would you return the same status/error codes?